cypress/support/pages/contactForm.pageObject.js

```diff
@@ -19,11 +19,11 @@ class ContactFormPageObject extends PageObject {
     return cy.contains('.btn', 'Send message');
   }
 
-  typeEmail(email) {
+  typeEmail() {
```

Comment:
The `typeEmail` method is missing the `email` parameter. You should pass the `email` parameter to this method and use it in the `type` function.